radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/epoch/EpochManager.java

```diff
@@ -307,7 +307,9 @@ private void processLedgerUpdate(LedgerUpdate ledgerUpdate) {
 	private void processEpochChange(EpochChange epochChange) {
 		// Sanity check
 		if (epochChange.getEpoch() != this.currentEpoch() + 1) {
-			throw new IllegalStateException("Bad Epoch change: " + epochChange + " current epoch: " + this.currentEpoch);
+			//TODO: looks like byzantine message, might be used for slashing?
```

Comment:
Why was this exception removed?

Reply:
Because it will crash the node. Throwing an exception means that we trust the event received from network, which is not the case.

Reply:
But it seems that's what we want to do in this case. @LukasGasior1, @talekhinezh?

Reply:
It seems two things can happen: either we received an invalid message from an honest node cause of an inconsistency in the consensus or an invalid message from a malicious node. How can we differentiate between them?